apps/ocean-api/testing/OceanApiTesting.ts

```diff
@@ -2,19 +2,20 @@ import { Testing, TestingGroup } from '@defichain/jellyfish-testing'
 import { OceanApiClient } from '@defichain/ocean-api-client'
 import { OceanStubServer } from './OceanStubServer'
 import { OceanStubClient } from './OceanStubClient'
+import { ApiTesting } from '../../libs/rootserver/testing/ApiTesting'
 import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
 import { ApiClient } from '@defichain/jellyfish-api-core'
-import { NestFastifyApplication } from '@nestjs/platform-fastify'
 
 /**
  * OceanApi Testing framework.
  */
-export class OceanApiTesting {
+export class OceanApiTesting extends ApiTesting {
   constructor (
-    private readonly testingGroup: TestingGroup,
-    private readonly stubServer: OceanStubServer = new OceanStubServer(testingGroup.get(0).container),
+    readonly testingGroup: TestingGroup,
```

Comment:
```suggestion
    private readonly testingGroup: TestingGroup,
```

As you do not refactor this line of code, you should not change the access modifier
Like https://github.com/JellyfishSDK/jellyfish/pull/1408/files#diff-5e255d386b0ec1ca0d1a2d057887be946d3db7f1024150e2bcf0ac2201165081L14

Reply:
Oops I changed it earlier but forgot to commit it, haha apologies and thanks for pointing it out!